run.py

```diff
@@ -447,6 +448,10 @@ def calculate_average_read_length(file_path):
                     )
                 )
                 rrna_reads_dict[inputs[0]] = subset_reads - non_rrna_count
+        if not file_present:
```

Comment:
How about `if not total_files_in_sample`, which means you don't need `file_present` at all?